docs/editors/atom.md

```diff
@@ -3,7 +3,23 @@ id: atom
 title: Atom
 ---
 
-Metals will soon work with Atom thanks to the
-[`ide-scala`](https://atom.io/packages/ide-scala) plugin. The `ide-scala` plugin
-is currently in the process of adding support for latest Metals version. Stay
-tuned for more updates.
+Metals work with Atom thanks to the
+[`ide-scala`](https://atom.io/packages/ide-scala) package.
+
+![Atom demo](assets/atom-demo.gif)
+
+```scala mdoc:requirements
+
+```
+
+## Installing the package
+
+Install the package by searching for "ide-scala" or run the following command.
+
+```sh
+apm install ide-scala
+```
```

Comment:
You can also link directly to atom://settings-view/show-package?package=ide-scala